test/e2e/app-dir/undefined-default-export/undefined-default-export.test.ts

```diff
@@ -1,16 +1,13 @@
 import path from 'path'
-import { FileRef, nextTestSetup } from 'e2e-utils'
+import { isNextStart, nextTestSetup } from 'e2e-utils'
 
 describe('Undefined default export', () => {
   const { next, isNextDev } = nextTestSetup({
-    files: new FileRef(
-      path.join(__dirname, 'fixtures', 'undefined-default-export')
-    ),
+    files: path.join(__dirname),
+    skipDeployment: true,
```

Comment:
you would need to use the `skipped` returned from `nextTestSetup` to fully skip the test in deployment

Reply:
Hm, looks like it actually [succeeded](https://github.com/vercel/next.js/actions/runs/18011450235/job/51248830200?pr=84240). I thought we can't access logs there, or is that only true for runtime logs, and not build logs?